src/main/java/ladder/view/ResultView.java

```diff
@@ -1,16 +1,66 @@
 package ladder.view;
 
-import ladder.domain.Ladder;
+import ladder.domain.*;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
 
 public class ResultView {
-    private final Ladder ladder;
+    public static final int START_INCLUSIVE = 0;
+    public static final String LINE_BREAK = "\n";
```

Comment:
JDK 에서 제공하는 개행 문자열을 사용해 보셔도 좋을 것 같아요
```suggestion
    public static final String LINE_BREAK = System.lineSeparator();
```

Reply:
기본 제공해주는 상수가 있었군요..😨

수정사항 반영해두겠습니다😅